webpack/containers/Application/overrides.scss

```diff
@@ -14,6 +14,10 @@ html .pagination-pf-pagesize.btn-group {
   margin-top: -6px !important;
 }
 
+td {
+  overflow: auto;
+}
+
```

Comment:
what is this for? Asking because it will affect all `td`'s

Reply:
So there is https://projects.theforeman.org/issues/25622 which is around the same issue.  I can remove this and have this tackled outside of this PR or leave it here if this indeed fixes it properly.